css/index.css

```diff
@@ -71,6 +71,36 @@ p {
     margin: 0 auto;
 }
 
```

Comment:
Great job commenting each of the sections!  It makes is a lot easier to find CSS.